src/main/java/com/aws/greengrass/dependency/Context.java

```diff
@@ -171,8 +171,6 @@ public Value getvIfExists(Object tag) {
         parts.compute(clazz, (tagObj, originalValue) -> {
             if (originalValue == null) {
                 originalValue = value;
-            } else {
```

Comment:
what if the value isn't null, don't we want to replace the value?

Reply:
Ideally we would not overwrite a clazz -> object mapping. Replaced compute with put since that is what we are doing.